osu/upload_test.py

```diff
@@ -0,0 +1,19 @@
+"""
+Pretty much just a sanity check to see if im uploading correctly
+"""
+import dropbox
+
+def upload_to_dropbox(file_from, file_to, access_token):
+    dbx = dropbox.Dropbox(access_token)
+
+    with open(file_from, "rb") as f:
+        dbx.files_upload(f.read(), file_to)
+
+    shared_link_metadata = dbx.sharing_create_shared_link_with_settings(file_to)
+    return shared_link_metadata.url
+
+access_token = ''
```

Comment:
please put this part in an if name is main thing otherwise it will run if acidentally imported